### DISCUSSION TITLE
Cron job

### DISCUSSION THREAD

**Discussion by fecqs** [vercel/community#2390] [Help]
### Summary

When the nextjs service starts, I want to execute some additional js scripts (such as Cron job). What should I do?

**Comment by amyegan** [comment]
Hi @fecqs! You can use Vercel CLI to create a [custom deployment workflow](https://vercel.com/guides/using-vercel-cli-for-custom-workflows) or set up a [Cron Job](https://vercel.com/docs/cron-jobs) on Vercel.
